addons/point_of_sale/models/pos_session.py

```diff
@@ -379,14 +384,20 @@ def _close_session_action(self, amount_to_balance):
             'context': {**self.env.context, 'active_ids': self.ids, 'active_model': 'pos.session'},
         }
 
-    def close_session_from_ui(self):
+    def close_session_from_ui(self, other_payment_method_diff_pairs=None):
         """Calling this method will try to close the session.
 
         If successful, it returns {'successful': True}
         Otherwise, it returns {'successful': False, 'message': str, 'redirect': bool}.
         'redirect' is a boolean used to know whether we redirect the user to the back end or not.
         When necessary, error (i.e. UserError, AccessError) is raised which should redirect the user to the back end.
         """
+
+        other_payment_method_diff_pairs = other_payment_method_diff_pairs or []
```

Comment:
as you are using  'or []' can't you just put "other_payment_method_diff_pairs=[]" in the signature and remove the or

Reply:
yup. I can do that.